swarm/network/stream/snapshot_sync_test.go

```diff
@@ -75,6 +75,7 @@ func dummyRequestFromPeers(_ context.Context, req *network.Request) (*enode.ID,
 //they are expected to store based on the syncing protocol.
 //Number of chunks and nodes can be provided via commandline too.
 func TestSyncingViaGlobalSync(t *testing.T) {
+	t.Skip("not working")
```

Comment:
Perhaps we shouldn't approve a PR that has skipped test without first clearing up why they are skipped?

Reply:
I did approve it because this test was being skipped from other PRs too (I am not sure actually what is the current status at all).

That's why I specifically ask for skipping in the context of **this** test. The reason could be related to other dependencies.

But yes, it is a contradiction to approve the PR and having such an important question unclear.